packages/nep141-erc20/src/bridged-nep141/sendToEthereum/index.js

```diff
@@ -584,23 +597,35 @@ async function checkSync (transfer) {
  */
 async function unlock (transfer) {
   const web3 = new Web3(getEthProvider())
-  const ethUserAddress = (await web3.eth.getAccounts())[0]
 
+  // Build burn proof
+  transfer = await checkSync(transfer)
+  const proof = await findProof(transfer)
+
+  if (await proofAlreadyUsed(web3, proof)) {
+    // TODO find the unlockTxHash (requires ERC20Locker upgrade)
```

Comment:
Please open an issue in https://github.com/aurora-is-near/rainbow-token-connector and let's add a link here to that issue

Reply:
https://github.com/aurora-is-near/rainbow-token-connector/issues/59